src/java.base/share/classes/jdk/internal/util/ClassFileDumper.java

```diff
@@ -134,48 +128,39 @@ public void dumpFailedClass(String name, byte[] bytes) {
 
     @SuppressWarnings("removal")
```

Comment:
Please remove this redundant suppression.

Reply:
Thanks, both fixed. IntelliJ IDEA keeps hiding these annotations, making them hard to spot.